models/Include/cps/DP/DP_Ph1_VoltageSource.h

```diff
@@ -54,12 +54,12 @@ namespace Ph1 {
 		void initializeFromNodesAndTerminals(Real frequency);
 		///
 		void setSourceValue(Complex voltage);
-		///
-		//void setSourceSignal(CPS::Signal::SignalGenerator::Ptr srcSig);
-		///
-		void setParameters(Complex voltageRef, Real srcFreq = 0.0);
+		/// Setter for reference voltage without creating a signal generator
+		void setParameters(Complex voltageRef);
+		/// Setter for reference voltage and frequency with a sine wave generator
+		void setParameters(Complex voltageRef, Real srcFreq);
```

Comment:
Why has the default value been removed here? I think this relates to a previous comment regarding setting this value.

Reply:
Putting a default value would collide with the other function setParameters(voltageRef) which does not create a signal generator. That one could be helpful if you want to create a voltage source of which the voltage is set externally during the simulation.